ddtrace/tracer/textmap.go

```diff
@@ -686,6 +686,10 @@ var (
 	// Equals character must be encoded with a tilde.
 	// Other disallowed characters must be replaced with the underscore.
 	originRgx = regexp.MustCompile(",|~|;|[^\\x21-\\x7E]+")
+
+	// validIDRegex is used to verify that the input is a valid hex string.
+	// The input must match the pattern from start to end.
```

Comment:
Let's be more specific about the input, i.e span ID? trace ID? both?

Reply:
Both, will add `validIDRegex is applicable for both trace and span ID.`